contracts/Comptroller.sol

```diff
@@ -12,7 +12,7 @@ import "./Governance/Comp.sol";
  * @title Compound's Comptroller Contract
  * @author Compound
  */
-contract Comptroller is ComptrollerV5Storage, ComptrollerInterface, ComptrollerErrorReporter, ExponentialNoError {
+contract Comptroller is ComptrollerV6Storage, ComptrollerInterface, ComptrollerErrorReporter, ExponentialNoError {
```

Comment:
Are you modifying the V5 storage? The point of this is that storage changes should be append only

Reply:
Nvm I see you added the V6 storage below, this was just the event change